routes/web.php

```diff
@@ -19,18 +19,19 @@
 
 Route::get('/home', 'HomeController@index')->name('home');
 
-//temporary dashoboard page rooting
 Route::get('/dashboard', function () {
     return view('dashboard');
 });
 
-//temporary admin_add_category page rooting
-Route::get('/admin_add_category', function () {
+
+Route::get('/admin/category/add', function () {
     return view('admin.addCategory');
 });
-
-//temporary routing for admin to add category 
-Route::post('/create_category', 'CategoryController@create');
+Route::get('/admin/category', 'CategoryController@index');
+Route::post('/admin/category/create', 'CategoryController@create');
+Route::get('/admin/category/edit/{category}', 'CategoryController@edit');
+Route::post('/admin/category/update/{category}', 'CategoryController@update');
+Route::get('/admin/category/{category}', 'CategoryController@delete');
```

Comment:
Let's change "Route::get('/admin/category/{category}', 'CategoryController@delete');" to "Route::get('/admin/category/delete/{category}', 'CategoryController@delete');" to stay uniformed with your implementation.